modules/claims_api/app/controllers/claims_api/base_disability_compensation_controller.rb

```diff
@@ -62,17 +62,5 @@ def track_526_validation_errors(errors)
         StatsD.increment STATSD_VALIDATION_FAIL_TYPE_KEY, tags: ["key: #{key}"]
       end
     end
-
-    def service(auth_headers)
-      if Settings.claims_api.disability_claims_mock_override && !auth_headers['Mock-Override']
```

Comment:
Don't remove this, This method isn't being used in the base controller because it is meant to be used in the above controllers that inherit from base controller. If I'm wrong show me but don't just remove this method.